corehq/apps/export/tests/test_views.py

```diff
@@ -129,21 +127,21 @@ def _create_case_export_instance(self):
 
 class TestPossibleGeoProperties(BaseExportViewTestCase):
 
-    def test_possible_geo_properties_toggle_disabled(self):
+    def test_possible_geo_properties_privilege_disabled(self):
         """
-        Test that _possible_geo_properties returns empty list when
-        toggle is disabled
+        Test that _possible_geo_properties returns empty list when the
+        domain does not have the required privilege
         """
         export_instance = self._create_form_export_instance_with_geopoint()
         view = get_base_export_view(export_instance, FORM_EXPORT)
-        with patch.object(toggles.SUPPORT_GEO_JSON_EXPORT, 'enabled') as mock_toggle:
-            mock_toggle.return_value = False
+        with patch('corehq.apps.export.views.new.domain_has_privilege') as mock_privilege:
+            mock_privilege.return_value = False
 
             result = view._possible_geo_properties
             self.assertEqual(result, [])
 
-    @flag_enabled('SUPPORT_GEO_JSON_EXPORT')
-    def test_possible_geo_properties_bulk_case_export(self):
+    @patch('corehq.apps.export.views.new.domain_has_privilege')
```

Comment:
I believe there is a `@privilege_enabled` decorator. maybe some of these tests can be rewritten to use that?

Reply:
:+1: https://github.com/dimagi/commcare-hq/pull/36874/commits/9b34d50f570ab7cbf5abfb2652513596ae590a3a